docs/source/release_notes/release-notes.rst

```diff
@@ -57,6 +57,10 @@ develop
            Please consult the documentation for :ref:`AtlasDB Configuration <atlas-config>` for details on how to set this up.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2263>`__)
 
+    *    - |improved|
+         - Added two new methods to Transaction, getUnfetchedRanges and a concurrent version of getRanges. These allow for better control over concurrency than the old getRanges method.
```

Comment:
Could you add some docs to tell other-product devs how they should be using said methods, please?